src/features/board/boardSlice.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from '../../app/store';
+
+export interface Card {name: string};
+
+const initialState: {player: Card[], opponent: Card[]} = {
+    player: [
```

Comment:
I feel like an anoying guy, but maybe it could be more explicit ? :3

Reply:
1 - You are allowed to be annoying when you make a comment on a PR
2 - I am not quite sure what you mean thought, do you have an example
3 - This is not code so we can talk french here no problem ! Maybe it would be easier to express ourselves ?

If this is just a naming issue for `player` just give it a name and we'll discuss it. But remember we are in the state board. So for me `board.player` is already explicit. Maybe we could name it playerCards and opponentCards, but since we use the interface `Card` to type initialState I felt like it was not necessary.

Tell me what you think about it :)

Reply:
Let's talk about it IRL ;)